packages/utils/telemetry-utils/src/sampledTelemetryHelper.ts

```diff
@@ -0,0 +1,131 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IDisposable, ITelemetryLogger, ITelemetryPerformanceEvent } from "@fluidframework/common-definitions";
+import { performance } from "@fluidframework/common-utils";
+
+interface Measurements {
+    latestDuration: number;
+    executionCount: number;
+    totalDuration: number;
+    minDuration: number | undefined;
```

Comment:
i would make these optional, this will prevent the key from existing when not being used, which will result in them not being part of the log if not being used when you rest expand the object

Reply:
```suggestion
    minDuration?: number;
```